File: source/Zapasovnik.API/Entities/Team.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Zapasovnik.API.Entities
{
    [Table("tbTeams")]
    public class Team
    {
        [Key]
        [Column("team_id")]
        public int TeamId { get; set; }

        [Column("team_name")]
        public required string TeamName { get; set; }

        [Column("team_established")]
        public DateTime TeamEstablished { get; set; }
    }
}

File: source/Zapasovnik.APP/app/src/main/java/com/example/zapasovnik/model/Team.kt
package com.example.zapasovnik.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Team (
    @SerialName("teamId")
    var TeamId: Int,

    @SerialName("teamName")
    var TeamName: String,

    @SerialName("teamEstablished")
    var TeamEstablished: String
)